src/query.rs

```diff
@@ -318,13 +318,13 @@ macro_rules! add_lints {
             )*
         }
 
-        fn get_query_text_contents() -> Vec<String> {
+        // No way to avoid this lint -- the push() calls are macro-generated.
+        #[allow(clippy::vec_init_then_push)]
```

Comment:
Is that a new clippy lint? Should I update my clippy?

Reply:
I don't think it's new, it's been there since 1.51 per this page: https://rust-lang.github.io/rust-clippy/master/index.html#vec_init_then_push

The old code didn't trigger that lint for me, though. Not sure why only the new code gets it.